### ISSUE TITLE
[aes/reseeding] DUT REPORTS status IDLE and INPUT_READY when in fact not ready

### ISSUE THREAD

**Issue by rasmus-madsen** [lowRISC/opentitan#10518]
reproduce 
get aes_sideload branch from my repo
sha 40ea78453ef2af47413c690299b1089effef504b
seed = 2261118925 
./util/dvsim/dvsim.py hw/ip/aes/dv/aes_sim_cfg.hjson -i aes_sideload -r 1 -t vcs -w=vpd -s=2261118925 --verbosity=m

When writing the CTRL shadow reg which requires two writes to update the DUT status register reads idle - while the dut idle output indicates that In fact the dut is busy, this leads to the second write being ignored- hence triggering an alert when a later write to this register is unexpected.

this results in a mismatch between the actual processed blocks and the expected blocks.
![image](https://user-images.githubusercontent.com/53917183/152027600-ed8bdc14-926a-4acd-91e4-747196cc39ed.png)

**Comment by rasmus-madsen** [comment]
@vogelpi if you want to use the mnemonic for the address bus like above.
save this in a file

```
ALERT_TEST 32'b????????????????????????000000?? #000000 #000000
KEY0_0 32'b????????????????????????000001?? #000000 #ffd7f3
KEY0_1 32'b????????????????????????000010?? #000000 #ffd700
KEY0_2 32'b????????????????????????000011?? #000000 #ffd700
KEY0_3 32'b????????????????????????000100?? #000000 #ffd700
KEY0_4 32'b????????????????????????000101?? #000000 #ffd700
KEY0_5 32'b????????????????????????000110?? #000000 #ffd700
KEY0_6 32'b????????????????????????000111?? #000000 #ffd700
KEY0_7 32'b????????????????????????001000?? #000000 #ffd700
KEY1_0 32'b????????????????????????001001?? #000000 #ffd700
KEY1_1 32'b????????????????????????001010?? #000000 #ffd700
KEY1_2 32'b????????????????????????001011?? #000000 #ffd700
KEY1_3 32'b????????????????????????001100?? #000000 #ffd700
KEY1_4 32'b????????????????????????001101?? #000000 #ffd700
KEY1_5 32'b????????????????????????001110?? #000000 #ffd700
KEY1_6 32'b????????????????????????001111?? #000000 #ffd700
KEY1_7 32'b????????????????????????010000?? #000000 #ffd700
IV0 32'b????????????????????????010001?? #000000 #ffd700
IV1 32'b????????????????????????010010?? #000000 #ffff00
IV2 32'b????????????????????????010011?? #000000 #ffff00
IV3 32'b????????????????????????010100?? #000000 #ffff00
DATA_IN0 32'b????????????????????????010101?? #000000 #ffff00
DATA_IN1 32'b????????????????????????010110?? #000000 #7fff00
DATA_IN2 32'b????????????????????????010111?? #000000 #7fff00
DATA_IN3 32'b????????????????????????011000?? #000000 #7fff00
DATA_OUT0 32'b????????????????????????011001?? #000000 #7fff00
DATA_OUT1 32'b????????????????????????011010?? #000000 #ff0000
DATA_OUT2 32'b????????????????????????011011?? #000000 #ff0000
DATA_OUT3 32'b????????????????????????011100?? #000000 #ff0000
CTRL 32'b????????????????????????011101?? #000000 #0000ff
ctrl_aux_shadow 32'b????????????????????????011110?? #000000 #00bfff
REGWEN 32'b????????????????????????011111?? #000000 #6a5acd
TRIGGER 32'b????????????????????????100000?? #000000 #00ffff
STATUS 32'b????????????????????????100001?? #000000 #11ff55
```
the file format is not important.
in the DVE wave right click and go to radix->user-defined->edit
click new 
add a name
set the check mark in table based
click import and find your file.
then click apply, and then ok.
 let me know if you have any trouble

**Comment by vogelpi** [comment]
Hey @rasmus-madsen ,
thanks for providing the details. I cannot reproduce it from your branch - the git hash changed. But I agree that this is a problem and I have an idea how to fix it. I am looking into this now.

**Comment by vogelpi** [comment]
Hey @rasmus-madsen ,
I've created a PR with a fix here: #10578

Can you please give this a try to see if it solves the issue?

**Comment by vogelpi** [comment]
This should have been fixed by #10578. If not, feel free to reopen it.
